src/components/ProductCard.jsx

```diff
@@ -0,0 +1,38 @@
+import placeholder from "../assets/placeholder.png";
+import heart from "../assets/heart.png";
+import styles from "./ProductCard.module.css";
+
+const ProductCard = ({ product }) => {
+  // 가격 포맷팅 (1000 -> 1,000)
+  const formatPrice = (price) => {
+    return price.toLocaleString("ko-KR");
+  };
+
+  return (
+    <div className={styles["best-product-item"]}>
+      <div className={styles["best-product-item-image"]}>
+        <img
+          src={product.images?.[0] || placeholder}
+          alt={product.name}
+          onError={(e) => {
+            e.target.src = placeholder;
+          }}
+        />
+      </div>
+      <div className={styles["best-product-item-info"]}>
+        <p className={styles["best-product-item-title"]}>{product.name}</p>
+        <p className={styles["best-product-item-price"]}>
+          {formatPrice(product.price)}원
```

Comment:
💊 제안
자주사용되는 함수의 경우 따로 정의하시는 것이 좋지만 지금과 같이 단순히 JS 내장 메소드를 호출하는 경우 아래처럼 바로 작성하시는 것이 더 가독성이나 효율성 면에서 좋을 것 같아요.


```suggestion
          {product.price.toLocaleString("ko-KR")}원
```